src/maud/posterior_predictive_model.stan

```diff
@@ -0,0 +1,371 @@
+functions{
```

Comment:
I think we should change the name of this file to `out_of_sample_model.stan` and add an explanatory comment